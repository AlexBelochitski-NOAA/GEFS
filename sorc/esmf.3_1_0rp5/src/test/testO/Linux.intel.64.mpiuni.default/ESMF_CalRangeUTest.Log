20190610 112232.676306 INFO     PET0 Running with ESMF Version 3.1.0rp5
20190610 112232.676563 INFO     PET0  Beginning Test, file ESMF_CalRangeUTest.F90, line          81
20190610 112232.676582 INFO     PET0  NUMBER_OF_PROCESSORS           1
20190610 112232.676600 INFO     PET0  Number of failed tests:           0
20190610 112232.676607 INFO     PET0  Ending Test, file ESMF_CalRangeUTest.F90, line         326
