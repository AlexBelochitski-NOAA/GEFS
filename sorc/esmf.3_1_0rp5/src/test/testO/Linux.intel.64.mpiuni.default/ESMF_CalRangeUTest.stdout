  Beginning Test, file ESMF_CalRangeUTest.F90, line          81
  NUMBER_OF_PROCESSORS           1
  Number of failed tests:           0
  Ending Test, file ESMF_CalRangeUTest.F90, line         326
